src/cpr_sdk/pipeline_general_models.py

```diff
@@ -84,6 +84,7 @@ class UpdateTypes(str, Enum):
     # DOCUMENT_STATUS = "document_status"
     METADATA = "metadata"
     REPARSE = "reparse"
+    REPROCESS = "reprocess"
```

Comment:
This would benefit significantly from some documentation adding here

Reply:
Deleting those commented update types would be really nice too, if you're happy with the drive by